python/private/pypi/BUILD.bazel

```diff
@@ -283,3 +283,27 @@ bzl_library(
     srcs = ["whl_target_platforms.bzl"],
     deps = [":parse_whl_name_bzl"],
 )
+
```

Comment:
I thought it would be good to evolve the repo towards a more standardised structure like the one described by [rules-template](https://github.com/bazel-contrib/rules-template)

Alternatively, if we would prefer to paint that bikeshed another time, I can hide as much of the implementation inside `//python/private/pypi` and we can restructure later.